manifests/external-dns-operator_v1_serviceaccount.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  creationTimestamp: null
```

Comment:
is the null creationTimestamp necessary in the manifest files?

Reply:
No, I don't quite know why but most of the Kube/OpenShift utilities add `null` creation timestamp when they scaffold/export things. That was added by operator-sdk when I used it to scaffold the OLM bundle.
I can remove it.

Reply:
interesting. sounds good.